tests/test_routedem.py

```diff
@@ -195,11 +197,13 @@ def test_routedem_d8(self):
                 'flow_accumulation_foo.tif',
                 'flow_direction_foo.tif',
                 'slope_foo.tif',
-                'stream_mask_foo.tif'):
+                'stream_mask_foo.tif',
+                'strahler_stream_order_foo.gpkg',
+                'subwatersheds_foo.gpkg'):

```

Comment:
Should we be testing these outputs at all besides for existence?

Reply:
Yeah, probably should, at least some cursory tests.  Added in 5dfa2e1a3